lib/spack/spack/test/stage.py

```diff
@@ -864,15 +867,14 @@ def test_diystage_preserve_file(self, tmpdir):
             _file.read() == _readme_contents
 
 
-@pytest.mark.skipif(sys.platform == "win32", reason="Not supported on Windows (yet)")
 def test_stage_create_replace_path(tmp_build_stage_dir):
     """Ensure stage creation replaces a non-directory path."""
     _, test_stage_path = tmp_build_stage_dir
     mkdirp(test_stage_path)
 
     nondir = os.path.join(test_stage_path, "afile")
     touch(nondir)
-    path = str(nondir)
+    path = url_util.path_to_file_url(str(nondir))
```

Comment:
What is necessitating this change?

Reply:
The first argument to stage expects a file url when it's a path. I'm not entirely sure how it was passing on linux but this helper was used to pass things into that first argument. Another instance where the same helper is used: https://github.com/spack/spack/blob/7bf3e9d5d7aef693a0c493662a7f2a15e40d3791/lib/spack/spack/test/stage.py#L416

Reply:
Sounds good!